runtime/src/main/java/org/corfudb/protocols/logprotocol/MultiObjectSMREntry.java

```diff
@@ -92,8 +91,11 @@ public void deserializeBuffer(ByteBuf b, CorfuRuntime rt) {
         for (int i = 0; i < numStreams; i++) {
             UUID streamId = new UUID(b.readLong(), b.readLong());
 
-            int updateLength = b.readInt();
-            byte[] streamUpdates = new byte[updateLength];
+            int start = b.readerIndex();
+            MultiSMREntry.seekToEnd(b);
```

Comment:
why it's a static method? Can we extract that to a utility class?
If not, I'm ok with current impl

Reply:
It can be. But since seeking logic is specific to a container, I feel like its appropriate to place here.